nodejs/eks/nodegroup.ts

```diff
@@ -150,12 +158,12 @@ export interface NodeGroupBaseOptions {
     /**
      * The tags to apply to the NodeGroup's AutoScalingGroup.
      */
-    autoScalingGroupTags?: { [key: string]: string };
+    autoScalingGroupTags?: pulumi.Input<InputTags>;
```

Comment:
Given that `InputTags` already includes an outer `pulumi.Input` - why do these also include that?

Reply:
Doh - thanks for the catch